File: zd1.kt
import java.util.Random

fun main() {
    
    var text = "qqqwwweeerrtsxcgfdfff"
    for (i in 0 until text!!.length - 2) {
        if (text[i] == text[i + 1] && text[i + 1] == text[i + 2]) {
            text = text.substring(0, i + 2) + randomChar() + text.substring(i + 3)
        }
    }
    println(text)
}

fun randomChar(): Char {
    val random = Random()
    val chars = "abcdefghijklmnopqrstuvwxyz".toCharArray()
    return chars[random.nextInt(chars.size)]
}

File: ZD1.js
function main() {
const readline = require('readline');
const rl = readline.createInterface({
input: process.stdin,
output: process.stdout
});

rl.question('Введите текст: ', (text) => {
for (let i = 0; i < text.length - 2; i++) {
if (text.charAt(i) === text.charAt(i + 1) && text.charAt(i + 1) === text.charAt(i + 2)) {
text = text.substring(0, i + 2) + randomChar() + text.substring(i + 3);
}
}
console.log(text);
rl.close();
});
}

function randomChar() {
const chars = "abcdefghijklmnopqrstuvwxyz";
return chars.charAt(Math.floor(Math.random() * chars.length));
}

main();
